internal/legacy/dockerregistry/inventory.go

```diff
@@ -692,7 +692,7 @@ func ParseContainerParts(s string) (
 		}
 		return strings.Join(parts[0:2], "/"), strings.Join(parts[2:], "/"), nil
 	default:
-		if parts[0] != "k8s.gcr.io" && parts[0] != "staging-k8s.gcr.io" {
+		if parts[0] != "registry.k8s.io" && parts[0] != "staging-k8s.gcr.io" {
```

Comment:
I'm wondering if we should keep k8s.gcr.io here, since it's kinda a valid registry. Or maybe it makes sense to drop it because of the registry freeze.

@puerco @upodroid What do you think about this?

Reply:
Does the registry freeze involve `<1.27` patch releases?

Reply:
needs more context on what this method is used for